src/components/label/styles.ts

```diff
@@ -0,0 +1,48 @@
+import {styled, Switch} from "@mui/material";
```

Comment:
Pretty weird naming for that file =) I assume it should be something like "theme-switch"